parsers/dortmund.py

```diff
@@ -0,0 +1,138 @@
+import re
+import datetime
+import urllib.request as rq
+
+from urllib.parse import urlencode as urlencode
+from bs4 import BeautifulSoup
+from utils import Parser
+
+from pyopenmensa.feed import LazyBuilder
+
+# dictionary for categories
+categories = {
```

Comment:
I put them in, because the websites are not that consistent, so that some times there is no alt/title tag set. What I did now is that the parser checks the title tag first and if it is empty it will use the dictionary.

Reply:
I see, however I still think that it is problematic to keep these kinds of lists in code, because they tend to be become outdated quickly. That being said, I am fine with leaving it in for now.